### PR TITLE
Move interfaces to src/Entity

### PR DESCRIPTION
**Pull request by pcambra**

### CODE REVIEW

File: modules/cart/src/CartProvider.php [hunk 1]
```diff
@@ -7,9 +7,8 @@
 
 namespace Drupal\commerce_cart;
 
-use Drupal\commerce_cart\CartSessionInterface;
```
Comment (bojanz) [inline modules/cart/src/CartProvider.php]: The CartSessionInterface is in use, there's no reason to remove it.

**Comment by bojanz** [comment]
Looks good, let's see what Travis says.

**Comment by bojanz** [comment]
PHP Fatal error:  Declaration of Drupal\commerce_store\StoreStorage::markAsDefault() must be compatible with Drupal\commerce_store\StoreStorageInterface::markAsDefault(Drupal\commerce_store\Entity\StoreInterface $store) in /home/travis/build/commerceguys/commerce/modules/store/src/StoreStorage.php on line 23

**Comment by bojanz** [comment]
Current failure: 

PHP Fatal error:  Interface 'Drupal\commerce_product\Entity\ProductVariationTypeInterface' not found in /home/travis/build/commerceguys/commerce/modules/product/src/Entity/ProductVariationType.php on line 45

**Comment by pcambra** [comment]
Come on! :D

**Comment by pcambra** [comment]
Not sure if there's something left

**Comment by bojanz** [comment]
Big thanks, Pedro :)

**Pull request merged**